tests/test_diffeq/test_ode/test_ivp_examples.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+
+import numpy as np
+
+from probnum.diffeq.ode import ivp, ivp_examples
+from probnum.random_variables import Constant
+from tests.testing import NumpyAssertions
+
+
+class TestExamples(unittest.TestCase, NumpyAssertions):
+    """Test cases for example IVPs: Lotka-Volterra, etc."""
+
+    def setUp(self):
+        self.tspan = (0.0, 4.212)
+
+    def test_logistic(self):
+        """Test the logistic ODE convenience function."""
+        rv = Constant(0.1)
+        lg1 = ivp_examples.logistic(self.tspan, rv)
+        self.assertEqual(issubclass(type(lg1), ivp.IVP), True)
```

Comment:
lets use `isinstance` instead of `issubclass(type`. I am aware that you didnt introduce it, but it would be great to have it fixed now that we're going over it :)

Reply:
Just as an addition: If you're writing a test you'll get an even more expressive output if you use `self.assertIsInstance(obj, cls)`, but otherwise what Nico said :).

Reply:
nice, thanks for the pointer! I should look into all the unittest assert options again at some point...

Reply:
If you want a quick overview, check out the sections in the guide: https://probnum.readthedocs.io/en/latest/development/unit_testing.html